Bottom: d8a9b592f0b0d57f1a8a8c67617bf5c0df0664a2
Top:    7adb9e874acaa19a22e0a4b3a351672617f52960
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-13 10:18:39 +0800

Refresh of autofs-5.0.7-make-dump-maps-report-nulled-entries.patch

---

diff --git a/lib/master.c b/lib/master.c
index 8277b36..7663d57 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1320,7 +1320,7 @@ int master_show_mounts(struct master *master)
 
 		printf("\nMount point: %s\n", ap->path);
 
-		/* Ignore all subsequent matching nulled entries */
+		/* Report nulled nulled entries */
 		if (cache_lookup_distinct(master->nc, path)) {
 			printf("\nmap entry %s nulled by previous entry\n",
 			       ap->path);
